packages/javascript/bh-shared-ui/src/components/HelpTexts/index.tsx

```diff
@@ -246,6 +247,8 @@ const EdgeInfoComponents = {
     SyncedToADUser: SyncedToADUser,
     SyncedToEntraUser: SyncedToEntraUser,
     CoerceAndRelayNTLMToSMB: CoerceAndRelayNTLMToSMB,
+    CoerceAndRelayNTLMToLDAP: CoerceAndRelayNTLMToLDAP,
+    CoerceAndRelayNTLMToLDAPS: CoerceAndRelayNTLMToLDAP,
```

Comment:
The LDAP and LDAPS edges share the same abuse text, which is why I'm simply re-using LDAP here